server/pages/index.js

```diff
@@ -318,13 +318,60 @@ function setUpLoggedInRoute( req, res, next ) {
 			res.redirect( redirectUrl );
 			return;
 		}
+
 		start = new Date().getTime();
 
 		debug( 'Issuing API call to fetch user object' );
-		user( req.cookies.wordpress_logged_in, geoCountry, function( error, data ) {
-			let searchParam, errorMessage;
 
-			if ( error ) {
+		user( req.cookies.wordpress_logged_in, geoCountry )
```

Comment:
We're not passing `callback` as an argument. Do we need to look at its use in [user-bootstrap/index.js](https://github.com/Automattic/wp-calypso/blob/refactor/rewrite-user-bootstrap-by-promise/server/user-bootstrap/index.js#L39)?

Reply:
The `callback` has been replaced by the `then` and `catch` handlers below, so we shouldn't need to worry about it anymore :)

Reply:
Okay, I was mainly wondering if we still need to catch that `API_KEY` Error, but now I see that that we're using `user-bootstrap` in another place in `pages/index.js`: https://github.com/Automattic/wp-calypso/blob/refactor/rewrite-user-bootstrap-by-promise/server/pages/index.js#L742